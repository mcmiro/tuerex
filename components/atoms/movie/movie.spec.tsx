import * as React from "react";
import {screen, render} from "@testing-library/react";
import Movie from "./";

describe("Test Movie component", () => {
	it("Should pass when file props in Movie is defined", () => {
		render(<Movie file="/test" data-testid="movie-test" />);
		const movie = screen.queryByTestId("movie-test");
		expect(movie).toBeDefined();
		expect(movie).toMatchSnapshot();
	});
});
