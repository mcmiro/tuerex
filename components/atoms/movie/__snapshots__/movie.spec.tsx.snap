// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Movie component Should pass when file props in Movie is defined 1`] = `null`;
